apps/gamify/web-app/src/app/pages/users/profile/profile.component.html

```diff
@@ -0,0 +1,164 @@
+                                                </div>
+
+                                                <div class="badge" matTooltip="Amount of gold badges obtained within this game">
+                                                    <mat-icon class="gold" aria-hidden="false" aria-label="Gold Badge Count">workspace_premium</mat-icon>
+                                                    <span>{{ userApp.badgesCount.GOLD }}</span>
+                                                </div>
+
+                                                <div class="badge" matTooltip="Amount of platinum badges obtained within this game">
+                                                    <mat-icon class="platinum" aria-hidden="false" aria-label="Platinum Badge Count">workspace_premium</mat-icon>
+                                                    <span>{{ userApp.badgesCount.PLATINUM }}</span>
+                                                </div>
+                                                
+                                            </div>
+                                        </div>
+
+                                        <div>
+                                            <span style="font-weight: 300">{{ userApp.obtainedBadgesCount }}/{{ userApp.totalBadgesCount }} Badges obtained</span>
+                                            <mat-progress-bar mode="determinate" [value]="userApp.obtainedPercentage"></mat-progress-bar>
+                                        </div>
+                                    </div>
+                                    
+                                </mat-expansion-panel-header>
+
+                                <div>
+                                    <mat-divider></mat-divider>
+
+                                    <h3 style="margin-bottom: 0">{{ userApp.obtainedBadgesCount }} Obtained badges</h3>
+
+                                    <div style="display: flex;">
+                                        <coders-badge *ngFor="let item of userApp.obtainedBadges | keyvalue" 
+                                            [tier]="item.value.badge.tier" 
+                                            [small]="true" 
+                                            [amount]="item.value.amount"
+                                            [name]="item.value.badge.name">
+                                        </coders-badge>
+                                    </div>
+                                    
+                                    <!-- <mat-divider style="margin-top: 5px;"></mat-divider>
+
+                                    <h3 style="margin-bottom: 0">5 Badges In progress</h3>
+
+                                    <div style="display: flex;">
+                                        <coders-badge *ngFor="let number of [0,1,2,3,4]" [tier]="badgeTier.GOLD" [small]="true"></coders-badge>
+                                    </div> -->
```

Comment:
Are you going to use this commented code?

Reply:
This is for future functionality, but since it isn't working yet there is no point in including it into the UI